Makefile

```diff
@@ -12,11 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# We need all the Make variables exported as env vars.
+# Note that the ?= operator works regardless.
+.EXPORT_ALL_VARIABLES:
```

Comment:
It would be helpful to note that this is specifically for `release-staging`. I'm not familiar with best practices here, but would it be preferable to explicitly export each individual variable instead so we don't end up with accidental exports in the future?

Reply:
Good point, I've removed and explicitly exported the required env vars.